lib/widgets/message_list.dart

```diff
@@ -255,12 +256,13 @@ class _MessageListPageState extends State<MessageListPage> implements MessageLis
     }
 
     return Scaffold(
-      appBar: AppBar(title: MessageListAppBarTitle(narrow: widget.narrow),
+      appBar: ZulipAppBar(
+        title: MessageListAppBarTitle(narrow: widget.narrow),
+        isLoading: store.isLoading,
         backgroundColor: appBarBackgroundColor,
         shape: removeAppBarBottomBorder
           ? const Border()
-          : null, // i.e., inherit
-      ),
+          : null),
```

Comment:
Let's keep this comment. A reader might see `null` and think it means there's no bottom border. That would be wrong; what `null` actually does is make the app bar inherit its bottom border from the `AppBarTheme`.